packages/config/config/devices/0x0005/pe653.json

```diff
@@ -15,45 +15,466 @@
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "40 °F",
+					"value": 40
+				},
+				{
+					"label": "41 °F",
+					"value": 41
+				},
+				{
+					"label": "42 °F",
+					"value": 42
+				},
+				{
+					"label": "43 °F",
+					"value": 43
+				},
+				{
+					"label": "44 °F",
+					"value": 44
+				}
+			]
+		},
+		{
+			"#": "50[0x010000]",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Freeze Protection: Circuit 1",
+			"description": "Turn on the circuit when system is in freeze protection mode. Requires connected air/freeze sensor.",
+			"valueSize": 4
+		},
+		{
+			"#": "50[0x020000]",
+			"$import": "#paramInformation/50[0x010000]",
+			"label": "Freeze Protection: Circuit 2"
+		},
+		{
+			"#": "50[0x040000]",
+			"$import": "#paramInformation/50[0x010000]",
+			"label": "Freeze Protection: Circuit 3"
+		},
+		{
+			"#": "50[0x080000]",
+			"$import": "#paramInformation/50[0x010000]",
+			"label": "Freeze Protection: Circuit 4"
+		},
+		{
+			"#": "50[0x100000]",
+			"$import": "#paramInformation/50[0x010000]",
+			"label": "Freeze Protection: Circuit 5"
+		},
+		{
+			"#": "50[0xff00]",
+			"label": "Freeze Protection: VSP Speed",
+			"description": "Pump speed while in freeze protection mode. Requires connected air/freeze sensor.",
+			"valueSize": 4,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 5,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "None",
+					"value": 0
+				},
+				{
+					"label": "VSP Speed 1",
+					"value": 2
+				},
+				{
+					"label": "VSP Speed 2",
+					"value": 3
+				},
+				{
+					"label": "VSP Speed 3",
+					"value": 4
+				},
+				{
+					"label": "VSP Speed 4",
+					"value": 5
+				}
+			]
+		},
+		{
+			"#": "50[0x80]",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Freeze Protection: Heater",
+			"description": "Turn on the heater when system is in freeze protection mode. Requires heater and connected air/freeze sensor.",
+			"valueSize": 4
+		},
+		{
+			"#": "50[0x7f]",
+			"label": "Freeze Protection: Pool/Spa Cycle Time",
+			"description": "Time to spend in each mode while cycling between pool and spa during freeze protection. Requires pool/spa configuration and connected air/freeze sensor.",
+			"valueSize": 4,
+			"unit": "minutes",
+			"minValue": 0,
+			"maxValue": 30,
 			"defaultValue": 0,
 			"unsigned": true
 		},
 		{
-			"#": "22",
-			"label": "Pool - Spa Mode",
-			"description": "Controller Mode",
-			"valueSize": 1,
-			"unit": "Decimal",
+			"#": "4",
+			"label": "Circuit 1 Schedule 1",
+			"description": "Start time (first two bytes, little endian) and stop time (last two bytes, little endian) of schedule in minutes past midnight, e.g. 12:05am (0x0500) to 3:00pm (0x8403) is entered as 83919875. Set to 4294967295 (0xffffffff) to disable.",
```

Comment:
These will be a lot of repeated and long descriptions. Maybe the other descriptions should just refer to this one?